app/forms/content_type_form.rb

```diff
@@ -18,7 +18,7 @@ class ContentTypeForm < ApplicationChangeSet
     'webarchive-seed' => Cocina::Models::ObjectType.webarchive_seed
   }.freeze
 
-  DIRECTIONS = %w[right-to-left left-to-right].freeze
+  DIRECTIONS = %w[left-to-right right-to-left].freeze
 
```

Comment:
Isn't "left-to-right" the more common option?  If so, shouldn't it be first in the drop down menu?

Reply:
verified with Andrew, swapping the order